src/Components/LandingPage/sections/RateSection.tsx

```diff
@@ -0,0 +1,119 @@
+
+import { Box, Grid } from '@mui/material';
+import Button from '@mui/material/Button';
+import Rating from '@mui/material/Rating';
+import Typography from '@mui/material/Typography';
+import { useEffect, useState } from "react";
+import ArrowCircleLeftOutlinedIcon from '@mui/icons-material/ArrowCircleLeftOutlined';
+import ArrowCircleRightOutlinedIcon from '@mui/icons-material/ArrowCircleRightOutlined';
+import axios from 'axios';
+
+export default function Rate() {
+
+  const [rate, setRate] = useState()
+  
+  const getReviews = () => {
```

Comment:
Don't forget to add loading state for this call, to indicates that reviews are being loaded.